client/components/table/placeholder.js

```diff
@@ -42,6 +51,9 @@ TablePlaceholder.propTypes = {
 			required: PropTypes.bool,
 		} )
 	),
+	/**
+	 * An integer with the number of rows to display. Defaults to 5.
```

Comment:
Defaults are added from the doc generator from `defaultProps`, so you can leave it off here. [See `Flag`'s `height`, for example.](https://woocommerce.github.io/wc-admin/#/components/flag?id=height)